src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -845,6 +863,8 @@ def load_arguments(self, _):
         c.argument('outfile', arg_type=bicep_outfile_type)
         c.argument('stdout', arg_type=bicep_stdout_type)
         c.argument('no_restore', arg_type=bicep_no_restore_type, help="When set, generates the parameters file without restoring external modules.")
+        c.argument('output_format', arg_type=CLIArgumentType(options_list=['--output-format'], help="Set output format. Valid values are ( json | bicepparam )."))
+        c.argument('include_params', arg_type=CLIArgumentType(options_list=['--include-params'], help="Set include params. Valid values are ( all | required-only )."))
```

Comment:
Please do not explicitly define the default option name repeatedly
```suggestion
        c.argument('output_format', help="Set output format. Valid values are ( json | bicepparam )."))
        c.argument('include_params', help="Set include params. Valid values are ( all | required-only )."))
```